### PR TITLE
Change to graphical user interface

### PR DESCRIPTION
**Pull request by wightfall**

### CODE REVIEW

**Comment by leibylucw** [comment]
@wightfall I'm currently working on a refactor of the patcher code itself on #22. I'm hoping to have that completed by the beginning of next week. All code and documentation will be updated, so you should have everything you'll need to introduce a GUI. You'll need to start from scratch as I have made changes to the patcher code.

I do ask, however, that when you eventually work on the repo, you please keep the GUI code separate from the patcher code. This makes the code more readable and easier to work with.

In summary, I'll let you know when the refactor code is merged into main. At that point, I'd suggest cloning the repo from scratch, setting it up, and opening a PR per the usual flow.

If you have any questions, let me know.

**Comment by wightfall** [comment]
I have one question if i make a gui do i need to separate in another file?

**Comment by leibylucw** [comment]
Yes. I'd like to enforce separation of concerns so that the code responsible for patching the game is independent from the code that builds a GUI. Tightly coupling code makes it hard to develop additional features down the road.

**Comment by leibylucw** [comment]
Closing as this PR is stale and the patch is about to undergo fundamental code changes.
